Added new resource psu:InstPol (#1070)
CiscoDevNet/terraform-provider-aci: website/docs/d/psuinstpol.html.markdown

---

> subcategory: -

Reviewer: ```suggestion
subcategory: "Fabric Policies"
```

layout: "aci"
page_title: "ACI: aci_power_supply_redundancy_policy"
sidebar_current: "docs-aci-data-source-power_supply_redundancy_policy"
description: |-
  Data source for ACI Power Supply Redundancy Policy
---

# aci_power_supply_redundancy_policy #

Data source for ACI Power Supply Redundancy Policy

## API Information ##

* `Class` - psuInstPol
* `Distinguished Name` - uni/fabric/psuInstP-{name}

## GUI Information ##

* `Location` - Fabric -> Fabric Policies -> Policies -> Switch -> Power Supply Redundancy

## Example Usage ##

```hcl
data "aci_power_supply_redundancy_policy" "example" {
  name  = "example"
}
```

## Argument Reference ##

* `name` - (Required) Name of the Power Supply Redundancy Policy object.

## Attribute Reference ##

> * `id` - Attribute id set to the Dn of the Power Supply Redundancy Policy.
> * `annotation` - (Read-Only) Annotation of the Power Supply Redundancy Policy object.
> * `name_alias` - (Read-Only) Name Alias of the Power Supply Redundancy Policy object.
> * `admin_rdn_m` - (Read-Only) Admin Redundancy Mode. The administrative state of the power supply policy.

Reviewer 2: There is some clean up being done and changes to auto-gen to create more consistency. Could you make few more doc changes that will follow this

```suggestion
* `id` - (Read-Only) Attribute id set to the Dn of the Power Supply Redundancy Policy. Type: String
* `annotation` - (Read-Only) Annotation of the Power Supply Redundancy Policy object. Type: String
* `name_alias` - (Read-Only) Name Alias of the Power Supply Redundancy Policy object. Type: String
* `admin_rdn_m` - (Read-Only) Admin Redundancy Mode. The administrative state of the power supply policy. Type: String
```

> * `admin_rdn_m` - (Read-Only) Admin Redundancy Mode. The administrative state of the power supply policy. Type: String

Reviewer 3: ```suggestion
* `administrative_state` - (Read-Only) Admin Redundancy Mode. The administrative state of the power supply policy. Type: String
```

---
subcategory: -
layout: "aci"
page_title: "ACI: aci_power_supply_redundancy_policy"
sidebar_current: "docs-aci-data-source-power_supply_redundancy_policy"
description: |-
  Data source for ACI Power Supply Redundancy Policy
---

# aci_power_supply_redundancy_policy #

> Data source for ACI Power Supply Redundancy Policy
> 
> 
> ## API Information ##

Reviewer 2: ```suggestion
Data source for ACI Power Supply Redundancy Policy

## API Information ##
```

* `Class` - psuInstPol
* `Distinguished Name` - uni/fabric/psuInstP-{name}

## GUI Information ##

> * `Location` - Fabric -> Fabric Policies -> Policies -> Switch -> Power Supply Redundancy
> 
> 
> ## Example Usage ##

Reviewer 2: ```suggestion
* `Location` - Fabric -> Fabric Policies -> Policies -> Switch -> Power Supply Redundancy

## Example Usage ##
```

```hcl

> data "aci_power_supply_redundancy_policy" "example" {
> 
>   name  = "example"
> }

Reviewer 2: ```suggestion
data "aci_power_supply_redundancy_policy" "example" {
  name  = "example"
}
```

```

## Argument Reference ##

* `name` - (Required) Name of the Power Supply Redundancy Policy object.

## Attribute Reference ##
* `id` - Attribute id set to the Dn of the Power Supply Redundancy Policy.

> * `annotation` - (Optional) Annotation of the Power Supply Redundancy Policy object.
> * `name_alias` - (Optional) Name Alias of the Power Supply Redundancy Policy object.
> * `admin_rdn_m` - (Optional) Admin Redundancy Mode. The administrative state of the power supply policy.

Reviewer 2: When data sources are set to computed they should be changed to Read-Only

```suggestion
* `annotation` - (Read-Only) Annotation of the Power Supply Redundancy Policy object.
* `name_alias` - (Read-Only) Name Alias of the Power Supply Redundancy Policy object.
* `admin_rdn_m` - (Read-Only) Admin Redundancy Mode. The administrative state of the power supply policy.
```

Reviewer 2: I know `annotation` and `name_alias` also have optional set from generic function but please set Read-Only in docs because they will be overwritten regardless. We should in time probably change the annotation/name_alias attribute adds for all data-sources.
